primitives/src/sentry.rs

```diff
@@ -359,6 +359,7 @@ pub mod campaign {
         pub creator: Option<Address>,
         /// filters the campaigns containing a specific validator if provided
         pub validator: Option<ValidatorId>,
+        pub is_leader: Option<bool>,
```

Comment:
- [ ] Please add documentation and explain the behavior:
`false` & `None` will result in no filtration, while `Some(true)` will filter Campaigns by the provided `ValidatorId` or the authenticated `Auth.uid`

Reply:
There are 2 cases where this query is taken into account:
with the query parameter `?validator`
or if an `Auth.uid` is provided (i.e. there is a auth header with token)

Please include the latter part.